docs/source/release-notes/v86-k2.md

```diff
@@ -0,0 +1,103 @@
+* Users can now set thumbnails that show in the Avatar app. (#145)
+* Add SL-like camera inspect function (hold Alt key) as default functionality. (#64, #188)
+* Add clicking/triggering on an avatar shows a temporary nametag. (#179)
+* Add an "Explore" app as a temporary go-to, along with associated "Explore" menu in Interface. These will be replaced by a modern implementation upon or after the Metaverse API (Grid Server) completion. (#190, #199, #205)
+* Add a "More" app that lists and installs community-made scripts and apps directly from a [GitHub repo](https://github.com/kasenvr/community-apps). (#187)
+* This takes longer than you might expect: Tons of revisions to brand/debrand, update links, docs, rehost, etc. in the Interface by various contributors. Thank you! (#9, #11, #19, #23, #34, #37, #38, #40, #44, #47, #50, #58, #70, #71, #75, #85, #89, #91, #99, #100, #105, #116, #177, #219)
+
+#### Graphics
+
+* Procedural vertex shader support added. (#15)
+* Procedural shaders no longer require an environment variable to be activated. Instead, use Developer > Render > Enable Procedural Materials. (#10)
+* Add support for GLTF alphaModes. (#15)
+* Fix opacityMapModes for GLTF. (#15)
+![](_images/0860_k2_opacity.png)
+* Add support to allow Material entities to specify face culling method, support GLTF doubleSided property. (#15)
+* Fix wearables not disappearing with avatar. (#15)
+
+#### Technicals
+
+* Updated CMake's SDL2 file to build correctly on Windows and Linux. (#3, #20, #21)
+* Fix audio failure due to missing VR. (#31)
+* VideoDecodeStats folder added to .gitignore. (#2)
+* Fix crash on secondary camera usage. (#32)
+* Fix QMetaObject::invokeMethod crash when trying to bind menu. (#39)
+* Add experimental Ubuntu 19.10 build support. (#53)
+* VCPKG modernization. (#53)
+* Fix Linux Building with `export HIFI_MEMORY_DEBUGGING=1`. (#54)
+* Fix avatar walk speed per avatar scaling. (#55)
+* Fix CMake build failing due to Public Key on VCPKG. (#56)
+* Fix thousands of 'is too small to hold all values' warnings on Linux. (#58)
+* Added Opus Audio Codec support. (#86)
+* Fix running scripts in Interface from being cleared occasionally. (#162)
+* Always display vertical scrollbar in Settings dialogs that scroll. (#113)
+* GitHub CI work-in-progress. (#51)
+* Added ArrayBuffer support to websockets. (#66)
+* Fix Clara.io type error issue. (#67)
+* Make default copy constructors explicit. (#104)
+* Fix updated TBB library hash in vcpkg portfile. (#222)
+
+
+## Server
+
+The numbers at the end of each item are the PR numbers in the Project Athena [GitHub repo](https://github.com/kasenvr/project-athena/pulls?utf8=%E2%9C%93&q=milestone%3Av0860-k2).
+
+#### General
+
+* Added environment variables to change domain server ports. (#15)
+* Fix server log spam `///`. (#175)
+* Add .deb and .rpm server packages, enabling deployment to Ubuntu and Amazon Linux systems. (Thanks, Heather!) (#101)
```

Comment:
This PR isn't actually merged. It needs review / consideration.

Reply:
This is true, but I wanted to credit since it's going to be used before merging. But I suppose the suggestion to remove does make sense in a release notes context!